core/src/main/java/cucumber/runtime/order/OrderTypeFactory.java

```diff
@@ -0,0 +1,28 @@
+package cucumber.runtime.order;
+
+import java.util.Map;
+
+import cucumber.runtime.CucumberException;
+
+public class OrderTypeFactory {

```

Comment:
This mapping belongs in `RuntimeOptions`. Additionally I think the different orders, could be turned into an enum that implements/extends the `OrderType` interface/class.